addons/mrp_zebra/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Zebra printers integration for MRP",
+    'summary': """
+        Zebra printers integration for mrp module""",
+    'description': """
+        MRP extention of ability to print labels on Zebra thermal printers.
+    """,
+    'author': "Odoo SA",
+    'website': "https://www.odoo.com",
+
+    # for the full list
+    'category': 'mrp',
+    'version': '1.0',
+
+    # any module necessary for this one to work correctly
+    'depends': ['stock_zebra', 'mrp'],
```

Comment:
Maybe it can just depend on mrp.  If I am not mistaken, it does not use anything from the stock_zebra module.